server/i18n/en/errors.json

```diff
@@ -98,6 +98,12 @@
     },
     "status": {
       "empty": "You must choose a status"
+    },
+    "turnaroundWorkingDayCount": {
+      "empty": "You must enter an expected turnaround time",
+      "invalid": "The expected turnaround time is not a valid number of days",
+      "expectedNumber": "The expected turnaround time is not a valid number of days",
+      "mustBeAtLeast1": "The expected turnaround time is not a valid number of days"
```

Comment:
Suggest double check on that these are the expect errors if you've not already. This is where it's easy to get out of sync with the API and not realise

Reply:
will do, thanks!